Shared/Tests/Get-IISModules.Tests.ps1

```diff
@@ -0,0 +1,157 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidOverwritingBuiltInCmdlets', '', Justification = 'Pester testing file')]
+[CmdletBinding()]
+param()
+BeforeAll {
+    $Script:parentPath = (Split-Path -Parent $PSScriptRoot)
+    . $Script:parentPath\IISFunctions\Get-IISModules.ps1
+
+    function LoadApplicationHostConfig {
+        [CmdletBinding()]
+        [OutputType("System.Xml.XmlNode")]
+        param(
+            [string]$Path
+        )
+
+        $appHostConfig = New-Object -TypeName Xml
+        try {
+            $appHostConfig.Load($Path)
+        } catch {
+            throw "Failed to loaded application host config file. $_"
+            $appHostConfig = $null
+        }
+        return $appHostConfig
+    }
+
+    function Get-AuthenticodeSignature {
```

Comment:
I believe this should be a `mock` not a `function`

Reply:
It should work both cases. Changed it to Mock.